libraries/src/application/Helper.php

```diff
@@ -7,14 +7,22 @@
  * @license     GNU General Public License version 2 or later; see LICENSE
  */
 
+namespace Joomla\Cms\Application;
+
 defined('JPATH_PLATFORM') or die;
 
+use \JFactory;
+use \JLog;
+use \JFilterOutput;
+use \JInstaller;
+use \JText;
+
 /**
  * Application helper functions
  *
  * @since  1.5
  */
-class JApplicationHelper
+class Helper
```

Comment:
Generally I agree with that thinking, I just personally dislike the thought of namespaced class names that end up as `Helper` or `Site`.  Even with `as` references, it just doesn't feel right to me.